lib/galaxy/selenium/navigates_galaxy.py

```diff
@@ -590,8 +591,7 @@ def perform_upload_of_pasted_content(self, paste_data, **kwd):
         self._perform_upload(paste_data=paste_data, **kwd)
 
     def _perform_upload(self, test_path=None, paste_data=None, ext=None, genome=None, ext_all=None, genome_all=None):
-        if not self.is_beta_history():
-            self.home()
+        self.home()
```

Comment:
Did you look into getting this to work without the `home()` call?  I don't think we actually want to do this in most testing, right?

Reply:
Well it was originally in there so I didn't have to deal with multiple selector options for what textbox to paste content into, which select2 drop down has the extension, etc... but even working around that there is something buggy in Galaxy right now. If you do an upload, close the dialog, re-open it and do a new upload - a lot of the time the history panel will not refresh with the new content. I confirmed this behavior by hand without selenium.

Is that known and is that just on sqlite?